codegen/model/template_data.go

```diff
@@ -52,7 +51,8 @@ type ProjectData struct {
 }
 
 func NewTemplateData(project v1.AutopilotProject, operator v1.AutopilotOperator, templates packr.Box) (*ProjectData, error) {
-	projectGoPkg := util.GetGoPkg()
+	//projectGoPkg := util.GetGoPkg()
+	projectGoPkg := "not implemented"
```

Comment:
This function is used other places, why remove it here?

Reply:
this is a deprecated function, will be removing in a follow up